ludwig/utils/dataframe_utils.py

```diff
@@ -0,0 +1,68 @@
+from typing import Optional
+
+import numpy as np
+import pandas as pd
+
+from ludwig.constants import DASK_MODULE_NAME
+from ludwig.utils.types import DataFrame
+
+
+def is_dask_lib(df_lib):
+    """Returns whether the dataframe library is dask."""
+    return df_lib.__name__ == DASK_MODULE_NAME
+
+
+def is_dask_backend(backend: "Backend"):  # noqa: F821
+    """Returns whether the backend's dataframe is dask."""
+    return backend.df_engine.df_lib.__name__ == DASK_MODULE_NAME
```

Comment:
very minor nit, but why not do this instead?
`is_dask_lib(backend.df_engine.df_lib)`

Reply:
Good point. Done.